.github/workflows/monodevelop.yml

```diff
@@ -11,16 +11,124 @@ on:
       - '*'
 
 jobs:
-  linux:
-
-    runs-on: ubuntu-20.04
+  ubuntu2004:  
+    runs-on: ubuntu-latest
+    container:
+      image: "ubuntu:20.04"
+    steps:
+    - name: Dependencies
+      run: |
+        apt-get update
+        apt-get install -y sudo        
+        sudo apt-get update
+        sudo apt-get install -y wget
+        wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+        sudo dpkg -i packages-microsoft-prod.deb
+        sudo apt-get install -y apt-transport-https
+        sudo apt-get update
+        sudo apt-get install -y dotnet-sdk-3.1
+        sudo apt-get install -y gnupg ca-certificates
+        sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+        echo "deb https://download.mono-project.com/repo/ubuntu stable-focal main" | sudo tee /etc/apt/sources.list.d/mono-official-stable.list
+        sudo apt-get update
+        sudo apt-get install -y sed build-essential intltool fsharp gtk-sharp2
+        sudo apt-get install -y software-properties-common
+        sudo add-apt-repository ppa:git-core/ppa -y
```

Comment:
the 20.04 build was working already without the need to depend on a PPA; I guess the PPA is needed only for the older ubuntu versions? if so, it's better to not add in this CI lane

Reply:
@lextm confirmed, please remove the ppa line from the 20.04 build